app/scripts/proactive/templates/job-variable-template.html

```diff
@@ -53,6 +53,9 @@ <h4 class="modal-title"><%- jobName %> </h4>
                     <label class="radio-inline"><input type="radio" name="<%- variable.Name %>" value="true"/>TRUE</label>
                     <label class="radio-inline"><input type="radio" name="<%- variable.Name %>" value="false" checked/>FALSE</label>
                 </div>
+            <% } else if (variable.hasOwnProperty("Model") && variable.Model.toLowerCase().indexOf('pa:hidden')!=-1) {  %>
+                <input class="variableValue textareavalues" style="border-radius: 4px; border: 1px solid #ccc; vertical-align: middle; color: #555;" name="<%- variable.Name %>" type="password" data-variable-model="<%- variable.Model %>" id="<%- key %>" value="<%- variable.Value %>" />
+                <label class="Extra" data-toggle="tooltip" title="<%- variable.Model %>"><%- variable.hasOwnProperty("Model") ? ((variable.Model.length >= 20) ? variable.Model.substring(0, 20) + " ..." : variable.Model):""  %></label>
```

Comment:
sorry. i have to ask. what is this `20` constant means here?

Reply:
I copied it from the code for all the other types of variable. It shows "..." when the variable Model definition is too long. (But I don't know why it was the constant `20`).